backend/server/dissectors/dissected_request_test.go

```diff
@@ -353,80 +353,23 @@ func TestParseBoolFlagFromQuery(t *testing.T) {
 }
 
 // Verify reading/converting from URL
-
-func TestParseStringFromURL(t *testing.T) {
```

Comment:
Because we're not handling the urlparam map anymore, these tests all broke. I still thought it was important to test `URLParam`, which is basically now a wrapper for the chi function `chi.URLParam(m.Request, key)`, but there's some error handling and coercible stuff. However, I tried a bunch of methods and none of them worked (ie creating a mock implementation, monkey patching, copying the Chi's custom context and applying that in a test etc). The only way I could get it to work was passing in a second argument to the URLParam which was either chi or my fake implementation, but it seemed kind of over the top to add in a second arg to the 95 uses with chi, when this test is basically a wrapper for another library function, but also happy to do that if we think it would be beneficial!

Reply:
added back in due to other changes